src/store/plugins/names.js

```diff
@@ -105,19 +105,20 @@ export default (store) => {
         const names = await Promise.all(
           accounts.map(({ address }) => Promise.all([
             getPendingNameClaimTransactions(address),
-            middleware.getNamesOwnedBy(address)
-              .then(({ active }) => active.map(({ info, name, hash }) => ({
-                createdAtHeight: info.activeFrom,
-                expiresAt: info.expireHeight,
-                owner: info.ownership.current,
-                pointers: info.pointers,
-                autoExtend: !!(
+            middleware.getNames({ owned_by: address })
+              .then(({ data }) => data.filter(({ active }) => active)
```

Comment:
I suggest to move the `.filter` to the next line to make the code more readable.